gpii/node_modules/preferencesServer/test/preferencesServerTests.js

```diff
@@ -580,11 +580,11 @@ var getFixtures = [
                 "gpii-default": {
                     "name": "Default preferences",
                     "preferences": {
-                        "http://registry.gpii.net/common/onscreenKeyboard": true,
                         "http://registry.gpii.net/common/initDelay": 0.120,
                         "http://registry.gpii.net/common/cursorSpeed": 0.850,
                         "http://registry.gpii.net/common/cursorAcceleration": 0.800,
                         "http://registry.gpii.net/common/mouseEmulationEnabled": true,
+                        "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
```

Comment:
@kaspermarkus Hmm, some were take directly from ISO 24751 (`cursor speed`, `cursor acceleration`); `initDelay` has a much longer name in the ISO standard, and `mouseEmulationEnabled` was created in Cloud4all. 
I have added these terms to the table at https://wiki.gpii.net/w/Cloud4all_Third_Pilot:_Common_Terms#Unmanaged_Common_Terms.